File: wxrtc/src/main/java/com/wx/rtc/WXCallListener.java
package com.wx.rtc;

public interface WXCallListener {
    void onError(int errCode, String errMsg);
    void onCallReceived(String callerId);
    void onCallCancelled(String streamId);
    void onCallBegin(String roomId, WXRTCDef.Role callRole);
    void onCallEnd(String roomId, WXRTCDef.Role callRole);
    void onUserReject(String streamId);
    void onUserNoResponse(String streamId);
    void onUserLineBusy(String streamId);
    void onUserJoin(String streamId);
    void onUserLeave(String streamId);
}

File: wxrtc/src/main/java/com/wx/rtc/WXCallListener.kt
package com.wx.rtc

interface WXCallListener {
    fun onError(errCode: Int, errMsg: String)
    fun onCallReceived(callerId: String)
    fun onCallCancelled(streamId: String)
    fun onCallBegin(roomId: String, callRole: WXRTCDef.Role)
    fun onCallEnd(roomId: String, callRole: WXRTCDef.Role)
    fun onUserReject(streamId: String)
    fun onUserNoResponse(streamId: String)
    fun onUserLineBusy(streamId: String)
    fun onUserJoin(streamId: String)
    fun onUserLeave(streamId: String)
}
